src/witness/svcompSpec.ml

```diff
@@ -16,6 +17,8 @@ let of_string s =
       NoDataRace
     else if global_not = "overflow" then
       NoOverflow
+    else if global_not = "no-termination" then
+      Termination
```

Comment:
This is still incorrect. The formula for termination is `CHECK( init(main()), LTL(F end) )`.